src/main/java/com/thecookiezen/presentation/ClusterController.java

```diff
@@ -71,6 +71,14 @@ public String addHost(@Valid DockerHost dockerHost, @RequestParam long clusterId
         return "redirect:/clusters/all";
     }
 
+    @RequestMapping(value = "clusters/node", method = RequestMethod.DELETE)
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2\.